caasp-environment.yaml

```diff
@@ -13,3 +13,13 @@ parameters:
   # hostnames won't be resolved.
   internal_net_cidr: 172.28.0.0/24
   dns_nameserver: 172.28.0.2
+  # cpi parameters
+  cpi_auth_url: https://192.168.24.26:5000/
```

Comment:
Example could be `https://api.keystone.example.net:5000/v3`

Reply:
done. outdated